app/src/main/java/com/example/concordia_campus_guide/Models/Direction.java

```diff
@@ -35,11 +42,11 @@ public void setEnd(LatLng end) {
         this.end = end;
     }
 
-    public TransportType getTransportType() {
+    public String getTransportType() {
         return transportType;
     }
 
```

Comment:
If these changes are to change the enums to StringDefs then we could add the annotations here too.